main.py

```diff
@@ -0,0 +1,6 @@
+def main():
+    pass
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
end of line is missing. There are IDE plugins which can help you manage it. I use is with vscode.

Reply:
is ? I couldn't seem to find it on the vscode marketplace. I'm having trouble finding a holistic process as a part of the ide that would format all files. for example toml, .gitignore and so on. it seems that i have to install a formatter for different types of files ? which seems kind of ridiculous... bloats your already bloated ide.

I would resort to using git pre commit hooks. would you say this is a good solution in this case ? assuming my writings above checkout. 

the con of using a pre-commit hook is that i would need to set it up for each project I work on.

Reply:
Apologies for pointing you to the wrong direction. I use the following [settings](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files).

Reply:
BTW, using pre-commit is a very good option not only for handling this issue